src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url('../images/star-active.svg');
+    }
+  }
+
+  &__star {
+    background-image: url('../images/star.svg');
+    background-position: center;
+    background-repeat: no-repeat;
+    width: $width-star;
+    height: $height-star;
```

Comment:
The background image URLs are hardcoded, which makes it less maintainable if the path changes. Consider using a variable for the image path so that it can be easily updated in one place if needed. Also, ensure that the `$width-star` and `$height-star` variables are defined in `variables.scss` as per the checklist.